### ISSUE TITLE
[MaxEnt pipeline] Changing ymin in the default bbox leads to an error in loadFromStac.R

### ISSUE THREAD

**Issue by frousseu** [GEO-BON/bon-in-a-box-pipelines#193]
Changing the bbox ymin = -1971146 to ymin = -1571146 in the default pipeline leads to an error in loadFromStac.R that seems to be related to the STAC catalog. Some values do not create problems, others do...

**Comment by JoryGriffith** [comment]
Yes I am getting an error too. When I input those values it gives this error (screenshot below) but when I change it slightly to -1571140 it works. @glaroc do you know what is happening?

![Image](https://github.com/user-attachments/assets/c2274cf8-0d17-447c-9e51-9bb6f9bd0f69)

**Comment by glaroc** [comment]
Dependency needs to be changed to gdalcubes==0.7.1

**Comment by JoryGriffith** [comment]
That doesn't fully fix the problem unfortunately, it is still not working with some layers and bounding boxes.
